imports/plugins/core/accounts/client/containers/auth.js

```diff
@@ -40,8 +40,8 @@ class AuthContainer extends Component {
       isLoading: true
     });
     const errors = {};
-    const username = email.trim();
-    const pword = password.trim();
+    const username = email === undefined ? "" : email.trim();
+    const pword = password === undefined ? "" : password.trim();
```

Comment:
should we consider `null` as well? if so, maybe use a truthy/falsey test instead? `email ? "" : email.trim()`

Reply:
Testing for equality with `undefined` is a no-no as the results aren't deterministic. Use `typeof` instead.